polling_stations/apps/data_collection/management/commands/import_fylde.py

```diff
@@ -3,41 +3,103 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E07000119"
-    addresses_name = "local.2019-05-02/Version 1/Fylde-Democracy_Club__02May2019.csv"
-    stations_name = "local.2019-05-02/Version 1/Fylde-Democracy_Club__02May2019.csv"
-    elections = ["local.2019-05-02"]
-    match_threshold = 98
+    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019fylde.CSV"
+    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019fylde.CSV"
+    elections = ["parl.2019-12-12"]
+    match_threshold = 95
```

Comment:
Tend to only use this if really sure! I had a bit of a scan and it looks like they're mostly 'st annes' vs 'st. annes' so that's a good use case. But it is easy to let other things sneak through!